app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -157,7 +157,7 @@ def update_vm_memory(vm, virtual)
     state = vm.attributes.fetch_path(:status, :state)
     if state == 'up'
       vm.update_memory(virtual, guaranteed, :next_run => true)
-      vm.update_memory(virtual, nil, :next_run => false)
+      vm.update_memory(virtual, nil)
```

Comment:
@jhernand does it make sense to toggle on ovirt versions +3.6? Or are we assuming the absence of this will always be `next_run => false` ?